openshift/templates/web/dc.yaml

```diff
@@ -141,9 +141,6 @@ objects:
       redir /Community-Contributed-Content/Go-Crond https://docs.developer.gov.bc.ca/reusable-services-list/#go-crond permanent
 
       # developer tools redirects
-      redir /Cloud-Pathfinder /Cloud-Pathfinder-Onboarding-Journey permanent
-      redir /Developer-Tools/Cloud-Pathfinder /Cloud-Pathfinder-Onboarding-Journey permanent
```

Comment:
I think the redirects are still needed incase someone has bookmarked the old link. I think it would be something like:
```
redir /Cloud-Pathfinder https://digital.gov.bc.ca/cloud/public/ permanent
redir /Developer-Tools/Cloud-Pathfinder https://digital.gov.bc.ca/cloud/public/ permanent
```